examples/ola-recorder.cpp

```diff
@@ -154,6 +170,41 @@ int VerifyShow(const string &filename) {
   }
 }
 
+
+/**
+ * Playback a recorded show
+ */
+int PlaybackShow() {
+  const string filename = FLAGS_playback.str();
+  if (FLAGS_verify_playback) {
+    // Verify the show and print a summary before running
+    string summary;
+    const int verified = VerifyShow(filename, &summary);
+    // Printing a newline first makes this output look better in interactive
+    // terminal logs.
+    OLA_INFO << endl << summary;
```

Comment:
Do you mean because it spaces each iteration? This will give some weird effects like a line stamp with no output. Can you give some example output to explain what you mean, or do you mean in a loop, in which case a ---- separator might look better (but perhaps only when looping)? Or printing a cout << endl but only if we're not logging to syslog.

Reply:
The summary is only ever printed for the entire run, not per iteration (since each iteration is the same, and total frame counts are likely more helpful than frame counts per iteration).

Example:
```
ola_recorder -l3 --playback test.show --iterations 5
examples/ola-recorder.cpp:206: 
------------ Summary ----------
For all (5) iterations:
Universe 1: 20 frames
Universe 2: 15 frames

Total frames: 35
Total playback time: 0.05 seconds
examples/ShowPlayer.cpp:330: ----- 4 iteration(s) remain -----
examples/ShowPlayer.cpp:333: ----- Waiting 0 ms before looping -----
examples/ShowPlayer.cpp:330: ----- 3 iteration(s) remain -----
examples/ShowPlayer.cpp:333: ----- Waiting 0 ms before looping -----
examples/ShowPlayer.cpp:330: ----- 2 iteration(s) remain -----
examples/ShowPlayer.cpp:333: ----- Waiting 0 ms before looping -----
examples/ShowPlayer.cpp:330: ----- 1 iteration(s) remain -----
examples/ShowPlayer.cpp:333: ----- Waiting 0 ms before looping -----
```

Without leading newline:
```
ola_recorder -l3 --playback test.show --iterations 5
examples/ola-recorder.cpp:206: ------------ Summary ----------
For all (5) iterations:
Universe 1: 20 frames
Universe 2: 15 frames

Total frames: 35
Total playback time: 0.05 seconds
examples/ShowPlayer.cpp:330: ----- 4 iteration(s) remain -----
examples/ShowPlayer.cpp:333: ----- Waiting 0 ms before looping -----
examples/ShowPlayer.cpp:330: ----- 3 iteration(s) remain -----
examples/ShowPlayer.cpp:333: ----- Waiting 0 ms before looping -----
examples/ShowPlayer.cpp:330: ----- 2 iteration(s) remain -----
examples/ShowPlayer.cpp:333: ----- Waiting 0 ms before looping -----
examples/ShowPlayer.cpp:330: ----- 1 iteration(s) remain -----
examples/ShowPlayer.cpp:333: ----- Waiting 0 ms before looping -----
```
...which seems a bit ghastly IMO

Reply:
I'm not too bothered either way, if you move the summary down so it's nicely formatted, it would be good to put a line saying "verification of FILENAME:" or something maybe, to give the log line some context. That's probably the only/one of the only multi-line log messages, hopefully it doesn't upset the syslog option.